omod/src/main/webapp/fragments/header.gsp

```diff
@@ -114,15 +114,16 @@
 
 </script>
 <header>
-    <div class="logo">
-        <a href="${ logoLinkUrl }">
-            <img src="${ logoIconUrl }"/>
-        </a>
-    </div>
```

Comment:
In addition to where you've placed the code, I would also place this in the else block starting at line 134 so that the logo shows up even if the page is configured not to use bootstrap.

Reply:
Understood